cli/index.js

```diff
@@ -306,28 +306,40 @@ function run(paths) {
   ).then(async values => {
     console.log(chalk.green(`Built ${values.length} documents.`));
     const titles = {};
-    // XXX Support locales!
-    const allTitlesFilepath = path.join(STATIC_ROOT, "titles.json");
-    try {
-      titles.titles = JSON.parse(await readFile(allTitlesFilepath, "utf8"));
-      console.warn(
-        `Updating ${Object.keys(allTitles).length} file ${allTitlesFilepath}`
-      );
-    } catch (ex) {
-      // throw ex;
-      console.warn(`Starting a fresh new ${allTitlesFilepath}`);
-      titles.titles = {};
-    }
+    const titlesByLocale = {};
+    let localeKey;
     values.forEach(built => {
-      titles.titles[built.uri] = built.doc.title;
+      localeKey = built.uri.split("/")[1];
```

Comment:
```suggestion
      const localeKey = built.uri.split("/")[1];
```
this is just a regular variable that is scoped inside this (fat arrow) function. No need to replace its value.